### PR TITLE
Make the Autocomplete emit change events

### PR DESCRIPTION
**Pull request by illright**

### CODE REVIEW

File: attractions/autocomplete/autocomplete.svelte [hunk 1]
```diff
@@ -39,6 +43,7 @@
     {maxOptions}
     {optionComponent}
     {...$$restProps}
+    on:change
```
Comment (aabounegm) [inline attractions/autocomplete/autocomplete.svelte:46]: You keep forgetting about the rename before forwarding)

Comment (illright) [reply attractions/autocomplete/autocomplete.svelte:46]: If by _rename_ you mean wrapping the value into an object, then yes, you are right, fixed it

Comment (aabounegm) [reply attractions/autocomplete/autocomplete.svelte:46]: I meant renaming the event to `nativeEvent`

Comment (illright) [reply attractions/autocomplete/autocomplete.svelte:46]: That one isn't native, it comes from the `AutocompleteField`

Comment (aabounegm) [reply attractions/autocomplete/autocomplete.svelte:46]: oh, GitHub didn't show all the lines so I didn't notice 😅

**Pull request merged**